Module not found: Error: Can't resolve './convert-source-map' -- upgrade from 0.26.4 to 0.27.2

<!-- Before creating an issue please make sure you are using the latest version of css-loader. -->

**Do you want to request a *feature* or report a *bug*?**
<!-- Please ask questions on StackOverflow or the webpack Gitter (https://gitter.im/webpack/webpack). Questions will be closed. -->
Possible bug.

**What is the current behavior?**
Errors when upgrading from 0.26.4 to 0.27.2

**If the current behavior is a bug, please provide the steps to reproduce.**
<!-- A great way to do this is to provide your configuration via a GitHub gist. -->
Upgrade from 0.26.4 to 0.27.2

**What is the expected behavior?**
Same configuration should work in both versions.

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as your webpack version, Node.js version and Operating System.**
Node v6.10.0
Webpack 2.2.1

I am getting the following error when upgrading from 0.26.4 to 0.27.2:
```
ERROR in ../~/css-loader/lib/css-base.js
Module not found: Error: Can't resolve './convert-source-map' in '/Users/jeff/WebstormProjects/admin/node_modules/css-loader/lib'
 @ ../~/css-loader/lib/css-base.js 56:24-55
 @ ../~/css-loader!../~/sass-loader/lib/loader.js!./client/css/_main.scss
 @ ./client/css/_main.scss
 @ ./client/pack.js

ERROR in ./client/css/_main.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Can't resolve './convert-source-map' in '/Users/jeff/WebstormProjects/admin/node_modules/css-loader/lib'
    at factoryCallback (/Users/jeff/WebstormProjects/admin/node_modules/webpack/lib/Compilation.js:259:39)
    at /Users/jeff/WebstormProjects/admin/node_modules/webpack/lib/NormalModuleFactory.js:234:19
    at onDoneResolving (/Users/jeff/WebstormProjects/admin/node_modules/webpack/lib/NormalModuleFactory.js:59:20)
    at /Users/jeff/WebstormProjects/admin/node_modules/webpack/lib/NormalModuleFactory.js:126:20
    at /Users/jeff/WebstormProjects/admin/node_modules/async/dist/async.js:3694:9
    at /Users/jeff/WebstormProjects/admin/node_modules/async/dist/async.js:359:16
    at iteratorCallback (/Users/jeff/WebstormProjects/admin/node_modules/async/dist/async.js:933:13)
    at /Users/jeff/WebstormProjects/admin/node_modules/async/dist/async.js:843:16
    at /Users/jeff/WebstormProjects/admin/node_modules/async/dist/async.js:3691:13
    at apply (/Users/jeff/WebstormProjects/admin/node_modules/async/dist/async.js:21:25)
    at /Users/jeff/WebstormProjects/admin/node_modules/async/dist/async.js:56:12
    at /Users/jeff/WebstormProjects/admin/node_modules/webpack/lib/NormalModuleFactory.js:121:22
    at onResolved (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/Resolver.js:70:11)
    at loggingCallbackWrapper (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at afterInnerCallback (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/Resolver.js:138:10)
    at loggingCallbackWrapper (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at Resolver.applyPluginsAsyncSeriesBailResult1 (/Users/jeff/WebstormProjects/admin/node_modules/tapable/lib/Tapable.js:181:46)
    at innerCallback (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/Resolver.js:125:19)
    at loggingCallbackWrapper (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at /Users/jeff/WebstormProjects/admin/node_modules/tapable/lib/Tapable.js:283:15
    at /Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:38:4
    at loggingCallbackWrapper (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at afterInnerCallback (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/Resolver.js:138:10)
    at loggingCallbackWrapper (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at Resolver.applyPluginsAsyncSeriesBailResult1 (/Users/jeff/WebstormProjects/admin/node_modules/tapable/lib/Tapable.js:181:46)
    at innerCallback (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/Resolver.js:125:19)
    at loggingCallbackWrapper (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at /Users/jeff/WebstormProjects/admin/node_modules/tapable/lib/Tapable.js:283:15
    at innerCallback (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/Resolver.js:123:11)
    at loggingCallbackWrapper (/Users/jeff/WebstormProjects/admin/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
```
Here is a gist with my webpack.config.js:
https://gist.github.com/jeff-kilbride/e58a6831546efeb43fef8fae6b2d1da6

Downgrading to 0.26.4 resolves the issue. I have checked and the `convert-source-map.js` file does exist in my `node_modules/css-loader/lib` directory. I thought it might be related to #447, but I already have the `fs: "empty"` in my config.

Is there something I need to change in my config for 0.27.x?

--- a/lib/convert-source-map.js
+++ b/lib/convert-source-map.js
@@ -7,8 +7,6 @@ var path = require('path');
 
 var commentRx = /^\s*\/(?:\/|\*)[@#]\s+sourceMappingURL=data:(?:application|text)\/json;(?:charset[:=]\S+?;)?base64,(?:.*)$/mg;
 var mapFileCommentRx =
-  //Example (Extra space between slashes added to solve Safari bug. Exclude space in production):
-  //     / /# sourceMappingURL=foo.js.map           /*# sourceMappingURL=foo.js.map */
   /(?:\/\/[@#][ \t]+sourceMappingURL=([^\s'"]+?)[ \t]*$)|(?:\/\*[@#][ \t]+sourceMappingURL=([^\*]+?)[ \t]*(?:\*\/){1}[ \t]*$)/mg
 
 function decodeBase64(base64) {

--- a/lib/css-base.js
+++ b/lib/css-base.js
@@ -9,7 +9,7 @@ module.exports = function() {
 	// return the list of modules as css string
 	list.toString = function toString() {
 		return this.map(function (item) {
-			const content = cssWithMappingToString(item);
+			var content = cssWithMappingToString(item);
 			if(item[2]) {
 				return "@media " + item[2] + "{" + content + "}";
 			} else {

--- a/package.json
+++ b/package.json
@@ -13,7 +13,6 @@
   ],
   "dependencies": {
     "babel-code-frame": "^6.11.0",
-    "convert-source-map": "^1.3.0",
     "css-selector-tokenizer": "^0.7.0",
     "cssnano": ">=2.6.1 <4",
     "loader-utils": "^1.0.2",

